README.md

```diff
@@ -26,7 +26,18 @@ You must have Docker Desktop installed for this to work, as the environment is d
 1. Once that has completed, type `docker compose up db -d`
 1. Once that has completed, type `docker compose run --rm web bundle exec rails db:setup`
 1. Once that has completed, type `docker compose up -d`
-1. In your browser, navigate to `https://localhost:3000`
+1. In your browser, navigate to `http://localhost:3000`
+
+### Other helpful commands
+1. To stop the app, type `docker compose stop`
+1. To restart after stopping, type `docker compose up -d`
```

Comment:
Is it a good idea to add `--build`?

Reply:
Not in this project. Since the image doesn't exist, it'll build it automatically. And since this is meant to be completed in 2 hours, I wouldn't expect there to ever be a need to pull down the latest version of this project and update the image.

And if this was a long running project, you can use versioning on the image name field to achieve a re-build. 

I know the layers will be cached so it'll still be quick, but it's not required.